src/_utils/_git_secret_tools.sh

```diff
@@ -608,19 +608,27 @@ function _assert_keyring_contains_emails {
   local emails=$3
   _assert_keyring_emails "$homedir" "$keyring_name" "$emails" 1 # 1 here means 'expect $emails in keyring'
 }
+
 function _assert_keyring_doesnt_contain_emails {
   local homedir=$1
   local keyring_name=$2
   local emails=$3
   _assert_keyring_emails "$homedir" "$keyring_name" "$emails" 0 # 0 here means 'don't expect $emails in keyring'
 }
 
+function _assert_keyring_contains_emails_at_least_once {
```

Comment:
I don't think that we need this proxy, we can just call `_assert_keyring_emails`

Reply:
I thought about this and went the route of being more explicit